internal/gatewayapi/securitypolicy.go

```diff
@@ -273,16 +273,20 @@ func (t *Translator) translateSecurityPolicyForRoute(
 	}
 
 	if policy.Spec.OIDC != nil {
-		oidc, err = t.buildOIDC(policy, resources)
-		if err != nil {
-			return err
+		if oidc, err = t.buildOIDC(policy, resources); err != nil {
+			errs = multierror.Append(errs, err)
 		}
 	}
 
-	// Apply IR to all relevant routes
 	// It can be difficult to reason about the state of the system if we apply
 	// part of the policy and not the rest. Therefore, we either apply all of it
 	// or none of it (when get errors when translating the policy)
+
+	if errs != nil {
```

Comment:
I dont think we should return early, maybe 1 feature is valid (e.g. `cors`) and one isn't (e.g. `jwt`), we should probably still
set those values but also return err for status feedback

Reply:
Would it be confusing if the `Accepted` status is `false`, but part of the policy gets applied?

Reply:
I think thats fine, because the applied policy sub field is not tied to the error